x/axelarnet/keeper/msg_server.go

```diff
@@ -491,46 +484,15 @@ func (s msgServer) RetryIBCTransfer(c context.Context, req *types.RetryIBCTransf
 func (s msgServer) RouteMessage(c context.Context, req *types.RouteMessageRequest) (*types.RouteMessageResponse, error) {
 	ctx := sdk.UnwrapSDKContext(c)
 
-	msg, ok := s.nexus.GetMessage(ctx, req.ID)
```

Comment:
shouldn't remove the pre-route validation?

Reply:
It's all still there but moved. Do you see any validation missing?

Reply:
I see it now in route message 👍